radiation_obs/README.md

```diff
@@ -1,9 +1,9 @@
 # Radiative fluxes data from CERES
 
-This artifact contains monthly mean top-of-the-atmosphere radiative fluxes from the 
-Clouds and Earth's Radiant Energy Systems (CERES) Energy Balanced and Filled (EBAF) from March 2000 to March 2023. 
+This artifact contains monthly mean top-of-the-atmosphere and surface radiative fluxes from the 
+Clouds and Earth's Radiant Energy Systems (CERES) Energy Balanced and Filled (EBAF) from March 2000 to October 2019. 
 The data includes incoming solar flux, upward shortwave and longwave radiative fluxes, and net radiative fluxes
 for all-sky and clear-sky conditions. The net radiative fluxes is positive downward. All fluxes are in W m-2.
-The data can be downloaded from the data repository [Will and Schneider 2024](https://data.caltech.edu/records/z24s9-nqc90).
+The data was downloaded from [CERES website](https://ceres.larc.nasa.gov/data/) in September 2024.
 
 License: Creative Commons Attribution 4.0 International
```

Comment:
I couldn't find license information on this website: https://ceres.larc.nasa.gov/data/. @Sbozzolo Could you find anything?

Reply:
By default, NASA data products are CC0

https://science.data.nasa.gov/license/